### PR TITLE
Scan sample bug fixes

### PR DESCRIPTION
**Pull request by D-Triveni**

### CODE REVIEW

**Review by krish2718 — APPROVED** [review]

File: samples/wifi/scan/README.rst [hunk 1]
```diff
@@ -35,16 +35,16 @@ Configuration
...
 
 .. note::
         You can also use ``menuconfig`` to enable the ``Scan type`` option.
 
 See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how to run ``menuconfig``.
 
-Configuration options
-=====================
-
 The following sample-specific Kconfig option is used in this sample (located in :file:`samples/wifi/scan/Kconfig`) :
```
Comment (richabp) [inline samples/wifi/scan/README.rst:48]:
```suggestion
The following sample-specific Kconfig options are used in this sample (located in :file:`samples/wifi/scan/Kconfig`) :
```

**Review by richabp — COMMENTED** [review]
One minor change apart from the offline  discussion on the Kconfig descriptions.

File: samples/wifi/scan/prj.conf [hunk 2]
```diff
@@ -49,5 +49,10 @@ CONFIG_WIFI_SCAN_BAND=y
 CONFIG_WIFI_SCAN_CHAN=y
 CONFIG_WIFI_SCAN_CHAN_LIST="2:1,6,11_5:36,44,48"
 
+CONFIG_WIFI_SCAN_SSID_FILT_SELECT=y
+#Configure the below option to select
```
Comment (krish2718) [inline samples/wifi/scan/prj.conf:53]: Space after `#`

**Review by krish2718 — APPROVED** [review]

File: samples/wifi/scan/Kconfig [hunk 3]
```diff
@@ -12,15 +12,15 @@ config WIFI_MAC_ADDRESS
 	string "WiFi MAC address"
 	default "F6:CE:36:00:00:01"
 	help
-		WiFi MAC address to be used by the sample. If not set, the sample will
+		Specifies the WiFi MAC address to be used by the sample. If not set, the sample will
```
Comment (kittydepa) [inline samples/wifi/scan/Kconfig:15]:
```suggestion
		Specifies the Wi-Fi MAC address to be used by the sample. If not set, the sample will
```

**Review by richabp — APPROVED** [review]

**Comment by D-Triveni** [comment]
Changes are covered in the following PR - https://github.com/nrfconnect/sdk-nrf/pull/12274.
Hence closing this PR.

**Pull request closed without merging**